frontend/src/components/admin/assessment-creation/AssessmentEditorHeader.tsx

```diff
@@ -142,7 +142,7 @@ const AssessmentEditorHeader = ({
                     name="Archive"
                     onClick={() => {
                       onClosePopover();
-                      handleArchive();
+                      onArchiveModalOpen();
```

Comment:
I don't think we need to be validating on archive, which is why I removed the validation check here!